src/controllers/itemdetailpage.js

```diff
@@ -146,6 +146,33 @@ define(['loading', 'appRouter', 'layoutManager', 'connectionManager', 'userSetti
             page.querySelector('.trackSelections').classList.remove('hide');
             select.setLabel(globalize.translate('LabelVersion'));
             var currentValue = select.value;
+
+            var resolutionNames = [];
+            var sourceNames = [];
+            mediaSources.forEach(function (v) {
+                if (v.Name.endsWith('p') || v.Name.endsWith('i')) {
```

Comment:
I think this would make Interlaced appear before Progressive, but meh, im not sure who would keep both I/P copies of the same quality lol

Reply:
Both very true points. I convert all my interlaced video into progressive and can only imagine many others do to for the improvement in visual quality.